tests/test_threadpoolctl.py

```diff
@@ -213,16 +231,17 @@ def test_openmp_nesting(nthreads_outer):
 
 
 def test_shipped_openblas():
-    all_openblases = [ctypes.CDLL(path) for path in libopenblas_paths]
-    original_num_threads = [blas.openblas_get_num_threads()
-                            for blas in all_openblases]
+    # checks that OpenBLAS effectively uses the number of threads requested by
+    # the context manager
+    original_info = _threadpool_info()
```

Comment:
```suggestion
    original_infos = _threadpool_info()
```
To be consistent with the previous one, it makes it clear it is the same thing.
Or maybe change it before to have the same name as in `ctx._original_info`.

Reply:
I went for info everywhere, see [https://blog.harwardcommunications.com/2010/11/09/information/](https://blog.harwardcommunications.com/2010/11/09/information/) :)

Reply:
Perfect!